File: JavaScript/Resursion/SumOfNums.js
const prompt = require('prompt-sync')({sigint:true});

const n = parseInt(prompt("Enter N: "));

const getSum = (N) =>{
    if(N === 0){
        return 0;
    }
    return getSum(N-1) + N ;
}
const Res = getSum(n);
console.log("Sum: "+Res);

File: C++ Language/Resursion/SumOfNums.cpp
#include<iostream>
using namespace std;

int getSum(int N){
    if(N == 0){
        return 0;
    }
    return getSum(N-1) + N;
}
int main(void){
    int n;
    cout <<"Enter N:  ";
    cin >> n;
    int Res = getSum(n);
    cout << "Sum : " << Res;
}